ci-operator/config/openshift/dpu-operator/openshift-dpu-operator-main.yaml

```diff
@@ -77,6 +77,11 @@ tests:
   commands: HOME=/tmp make test
   container:
     from: src
+- as: e2e-dpu-operator-tests
+  cluster: build05
+  steps:
+    workflow: dpu-operator-workflow
```

Comment:
Can we give the "workflow" a better name? Let's say "dpu-operator-e2e-tests-workflow"?

Reply:
ack